content/en/blog/_posts/2024-03-08-mid-cycle-1.30.md

```diff
@@ -0,0 +1,121 @@
+implementation that addresses this problem by building a framework to support making these claim
+parameters less opaque. Instead of handling the semantics of all claim parameters themselves,
+drivers could manage resources and describe them using a specific "structured model" pre-defined by
+Kubernetes. This would allow components aware of this "structured model" to make decisions about
+these resources without outsourcing them to some third-party controller. For example, the scheduler
+would be able to allocate claims rapidly, without back-and-forth communication with dynamic resource
+allocation drivers. Work done for this release centers on defining the framework necessary to enable
+different "structured models" and to implement the "named resources" model. This model allows
+listing individual resource instances and, compared to the traditional device plugin API, adds the
+ability to select those instances individually via attributes.
+
+### Node memory swap support ([KEP-2400](https://kep.k8s.io/2400))
+
+In Kubernetes v1.30, memory swap support on Linux nodes gets a big change to how it works - with a
+strong emphasis on improving system stability. In previous Kubernetes versions, the `NodeSwap`
+feature gate was disabled by default and, when enabled, it used `UnlimitedSwap` behavior as the
+default behavior. To achieve better stability, `UnlimitedSwap` behavior (which might compromise node
+stability) will be removed in v1.30.
+
+The updated, still-beta support for swap on Linux nodes will be available by default, although the
+default behavior will be to run the node set to `NoSwap` (not `UnlimitedSwap`) mode. In `NoSwap`
+mode, the kubelet supports running on a node where swap space is active, but Pods don't use any of
+the page file. You'll still need to set `--fail-swap-on=false` for the kubelet to run on that node.
+However, the big change is the other mode: `LimitedSwap`. In this mode, the kubelet actually uses
+the page file on that node and allows Pods to have some of their virtual memory paged out.
+Containers (and their parent pods)  do not have access to swap beyond their memory limit, but the
+system can still make use of the swap space if available.
+
+Kubernetes' Node special interest group (SIG Node) will also update the documentation to help you
+understand how to use the revised implementation, based on feedback from end users, contributors,
+and the wider Kubernetes community.
+
+For more details on Linux node swap support in Kubernetes, read the previous [blog
+post](/blog/2023/08/24/swap-linux-beta/) or the [node swap
+documentation](/docs/concepts/architecture/nodes/#swap-memory).
+
+### Support user namespaces in pods ([KEP-127](https://kep.k8s.io/127))
+
+[User namespaces](/docs/concepts/workloads/pods/user-namespaces) is a Linux-only feature to better
+isolate pods to prevent or mitigate several CVEs rated high/critical, including the just published
```

Comment:
```suggestion
[User namespaces](/docs/concepts/workloads/pods/user-namespaces) is a Linux-only feature that better
isolates pods to prevent or mitigate several CVEs rated high/critical, including the just-published
```